molr-mole-core/src/main/java/io/molr/mole/core/logging/MissionLogHandler.java

```diff
@@ -0,0 +1,31 @@
+package io.molr.mole.core.logging;
+
+import io.molr.commons.domain.MissionHandle;
+import io.molr.commons.domain.MissionLog;
+import reactor.core.publisher.Flux;
+
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+
+// TODO Add javadoc
+public class MissionLogHandler {
```

Comment:
Is there no other way to handle this concept than by static methods? (Singletons are evil ;-)